6.3.longCatIsLong.py

```diff
@@ -0,0 +1,6 @@
+def count_words(sentence):
+    # This line removes any punctuation marks from the sentence and converts all words to lowercase
+    remove_words = [word.strip(',.:;!?').lower() for word in sentence.split() if word.strip(',.:;!?').isalpha()]

```

Comment:
',.:;!?' don't cover all of the possible signs that can appear beside letters.
Think of a better method to filter out these signs.